apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -181,6 +184,12 @@ def prepare_tag(html_tags, index, group_info={"tag": None, "depends": None}):
         answer_indent = pl.get_integer_attrib(html_tags, "indent", None)
         inner_html = pl.inner_html(html_tags)
         ranking = pl.get_integer_attrib(html_tags, "ranking", -1)
+        distractor_info = pl.get_string_attrib(element, "distractor-info", DISTRACTOR_INFO_DEFAULT)
```

Comment:
It looks like you're using the string `"none"` as a sentinel value. Instead, you should use the keyword `None` as the default sentinel, then your comparisons will be `distractor_info is None`.

Reply:
resolved by the fact that we got rid of this options altogether